docs/concepts/configuration/assign-pod-node.md

```diff
@@ -206,6 +206,31 @@ If defined but empty, it means "all namespaces."
 All `matchExpressions` associated with `requiredDuringSchedulingIgnoredDuringExecution` affinity and anti-affinity
 must be satisfied for the pod to schedule onto a node.
 
+#### More Practical Usage
+
+Than directly using with pods Interpod Affinity and AnitAffinity will be more useful when they are used with higher
+level objects such as Statefulset, Deployments, etc.  One can easily configure and prefer if two workloads should
```

Comment:
One can easily configure that a set of workloads should be co-located in the same defined topology, e.g., the same node.